patches/master_patch.patch

```diff
@@ -2025,6 +2025,73 @@ index d466ccd1e02ed8c158fda545fe2175c135018da5..16ae2a357b21de66b1558939f9333b3c
      }
    ]
  }
+diff --git a/third_party/wtl/include/atldlgs.h b/third_party/wtl/include/atldlgs.h
+index 62db9178013cd6e61cf763e11c079f388e20f144..85e5e6690d3b9429e381ad543203f507974d0549 100644
+--- a/third_party/wtl/include/atldlgs.h
```

Comment:
@ltilve - what error is this causing? I'd prefer not to add a patch for this if possible

Reply:
but these are all electron files, can't we just fix the file dialogs?

Reply:
Those files will eventually be removed in https://github.com/brave/muon/pull/385
Do we need to do extra effort on fixing the file dialog implementation which will not be there in the future?

Reply:
I agree that if those dialogs are being changed we can remove this fix once the PR is merged (otherwise the Win build would break).
I had also submitted a couple of days ago the fix to chromium upstream https://chromium-review.googlesource.com/c/chromium/src/+/880626 to avoid having to patch it ourselves if merged.

Reply:
looks like this is fixed with @darkdh's PR

Reply:
I'm going to back this one out and merge #385 into C64 for master